go.mod

```diff
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/dimiro1/banner v1.1.0
+	github.com/emvi/null v1.3.1
```

Comment:
I found this package in the awesome go's list and it seemed simple and lightweight. I used it internally, so (I guess) it doesn't affect the clients, because I generated the communication structs from proto file. 
My opinion about volatiletech's null is, this package had been created for sqlboiler and commonly for data persisting and I only wanted to use something for internal data transfer. So volatiletech/null should get us a bigger overhead than emvi/null.